BrowserKit/Sources/OnboardingKit/Views/OnboardingMultipleChoiceCardView.swift

```diff
@@ -0,0 +1,107 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import ComponentLibrary
+
+public struct OnboardingMultipleChoiceCardView<VM: OnboardingCardInfoModelProtocol>: View {
+    @State private var textColor: Color = .clear
+    @State private var cardBackgroundColor: Color = .clear
+    private var shadowColor = Color.black.opacity(UX.CardView.shadowOpacity)
```

Comment:
Normally colors are themed as well and not just a black color with opacity. I would check with the designer to make sure they use `shadowSubtle`, `shadowDefault` or `shadowStrong`

Reply:
Taking a second look I noticed shadow is not necessary anymore.